src/DotNetCore.CAP.RabbitMQ/CAP.RabbiMQOptions.cs

```diff
@@ -104,5 +116,50 @@ public class QueueArgumentsOptions
             // ReSharper disable once InconsistentNaming
             public int MessageTTL { get; set; } = 864000000;
         }
+                
+        public class BasicQos
+        {
+            /// <summary>
+            /// New intance of BasicQos sets the use of basic qos setup on the channel.
+            /// </summary>
+            /// <param name="prefetchCount">Sets the PrefetchSize.</param>
+            /// <param name="global">Sets Global flag (default false).</param>
+            public BasicQos(ushort prefetchCount, bool global = false)
+            {
+                PrefetchCount = prefetchCount;
+            }
+
+            /// <summary>
+            /// New intance of BasicQos sets the use of basic qos setup on the channel.
+            /// </summary>
+            /// <param name="prefetchSize">Sets the PrefetchSize.</param>
+            /// <param name="prefetchCount">Sets the PrefetchCount.</param>
+            /// <param name="global">Sets Global flag (default false).</param>
+            public BasicQos(uint prefetchSize, ushort prefetchCount, bool global = false)
+            {
+                PrefetchSize = prefetchSize;
+                PrefetchCount = prefetchCount;
+                Global = global;
+            }            
+
+            /// <summary>
+            /// Gets the PrefetchSize, a value of 0 is treated as infinite, allowing any number of unacknowledged message to be prefetched on server. 
+            /// The default value is 0.
+            /// </summary>
+            public uint PrefetchSize { get; private set; } = 0;
```

Comment:
Hi @nunorelvao ,

I noticed that the prefetch_size option is not implemented in RabbitMQ, I think we can remove it.

https://groups.google.com/g/rabbitmq-users/c/iiv_HPB-nqI

Reply:
https://www.rabbitmq.com/specification.html

![image](https://user-images.githubusercontent.com/8205994/217495186-94d39316-8658-4c9a-9714-e3997372f7c0.png)

Reply:
Yes you right and agreed , making this change to remove this option.